### PR TITLE
Fix logout consent issue in sample reactjs app

### PR DESCRIPTION
**Pull request by chaminjay**
https://user-images.githubusercontent.com/90854808/159432110-5bbcf7b8-b65a-433d-89c2-15ba984e17eb.mov

### CODE REVIEW

**Review by thivi — APPROVED** [review]

File: samples/asgardeo-html-js-app/app.js [hunk 1]
```diff
@@ -60,6 +60,16 @@ authClient.on("sign-out", function (response) {
     updateView();
 });
 
+/* 
+*   handles the error occurs when the logout consent page is enabled
+*   and the user clicks 'NO' at the logout consent page
```
Comment (Wathsara) [inline samples/asgardeo-html-js-app/app.js:65]: Shall we format the comment? Check the other places as well

Comment (chaminjay) [reply samples/asgardeo-html-js-app/app.js:65]: Thanks for the suggestion and will make sure to format comments in future PRs.
This PR would be obsolete when new proposed changes are made as mentioned in [#issuecomment-1096495048](https://github.com/wso2-enterprise/asgardeo-product/issues/3056#issuecomment-1096495048) and [#issuecomment-1102165798](https://github.com/wso2-enterprise/asgardeo-product/issues/3056#issuecomment-1102165798). Therefore these changes will no longer exist.

Comment (brionmario) [reply samples/asgardeo-html-js-app/app.js:65]: Are the new changes onboarded @chaminjay. If, so lets link the PR here.

Comment (chaminjay) [reply samples/asgardeo-html-js-app/app.js:65]: New changes added by https://github.com/asgardeo/asgardeo-auth-spa-sdk/pull/99

**Pull request merged**